src/Renderer/ThreeExtented/FeatureToolBox.js

```diff
@@ -0,0 +1,559 @@
+/**
+ * Generated On: 2016-09-28
+ * Class: FeatureToolBox
+ * Description:
+ */
+
+import THREE 					from 'THREE';
+import CVML 					from 'Core/Math/CVML';
```

Comment:
We can use earcut for triangulation : see issue #65 where we had this discussion.